VSRAD.Package/Commands/ActionsMenuCommand.cs

```diff
@@ -110,19 +112,28 @@ private async Task ExecuteActionAsync(ActionProfileOptions action)
         {
             try
             {
+                if (_actionRuns)
+                {
+                    Errors.Show(new Error($"Action {action.Name} is running.\nIf you think that this may be hang, please use Disconnect button."));
```

Comment:
There's no need for extra boxing, `Errors.ShowWarning(string)` is identical in this context.

Also I'd slightly alter the message to include `_currentActionName` instead of `action.Name`, like:
 "Action {_currentActionName} is already running.\nIf you believe this to be a hang, use the Disconnect button available in Tools -> RAD Debug -> Options to abort the currently running action."

Reply:
Thanks for the suggestion!
Resolved by 380d70c, c591575 and 1d60def.